TelerikBlazor.Server/DataAccess/NorthwindContext.cs

```diff
@@ -36,7 +33,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             if (!optionsBuilder.IsConfigured)
             {
                 //To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
-                optionsBuilder.UseSqlServer("Server=(LocalDb)\\MSSqlLocalDb;Database=Northwind;Trusted_Connection=True;");
+                optionsBuilder.UseSqlServer(@"Server=JIVANOVA\SQLEXPRESS;Database=Northwind;Trusted_Connection=True;");
```

Comment:
Can we remove the personal cridentials? Is the readme updated to note users that they should replace this string?

Reply:
it's already removed